docs/design/mockups/phase-4-homepage/round-r1-5b-hero-flourishes-hybrid-comparisons.html

```
@@ -0,0 +1,713 @@
+      <section class="candidate var-event" id="event">
+        <header>
+          <div>
+            <div class="key">R1.5b · Variant 3 of 4 · articleType = event</div>
+            <div class="name">Event hero · ported + day-block kept</div>
+            <div class="desc">
+              The event hero gets ONE kept overlay: the day-block stamp
+              that telegraphs "this is an event" before the visitor reads
+              anything. It earns its place because event articles have a
+              date the visitor cares about (am I free that day?) more than
+              any other variant. Below the H1, a compressed venue strip
+              ports the most-essential `<EventStrip>` row to the homepage
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </eventstrip> ], start tag match failed [ <EventStrip> ] on line 584.

(tag-pair)